README.md

```diff
@@ -1,2 +1,111 @@
-# Math-magicians
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  Make simple calculations. Read a random math-related quote
```

Comment:
- [ ] The README is missing the project title and description. I can see you have deleted this. Please add this to your README to make it professional.

Reply:
I actually have the project description in the "about" section of my repository, that would be redundant, right? But anyway, I will put the information.

![](https://media0.giphy.com/media/Daa5O9S8WuWYUZBucr/giphy.gif)

Reply:
Hi @xJokeer,

I can't see any about section in your readme file in the setup-quotes branch. And also make sure that putting the project title and description in the top of the readme is professional. Since many user come and will check the readme in future. 🙂

Reply:
![image](https://user-images.githubusercontent.com/74747182/133163921-bc61ebf2-3d4e-4645-8e65-894a1fbc5c8e.png)
 
Thanks for the review, I'll implement that on a better way😁